composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "dominik/dominik",
+    "name": "thekeymaster/dokuwiki-plugin-findologic-xml-export",
```

Comment:
Namespace should be `findologic`.

Reply:
TODO:

- Change `thekeymaster` to `findologic`.